### ISSUE TITLE
a fix for prePathURL of twisted/web/server.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7589]
|<img alt="rgl's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rgl reported|
|-|-|
|Trac ID|trac#310|
|Type|defect|
|Created|2003-10-02 07:09:32Z|

Attachments:

* [server.py.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/af5/af53d4aa0b9131f18f84130767ee5b1dcbcb63be/c6366a7e60df4b44a79cf242265e07fae3435708.patch) (365 bytes) - added by rgl on 2003-10-02 07:09:32Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__310 310
type__defect defect
reporter__rgl rgl
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1065078572000000 1065078572000000
changetime__1468113790156504 1468113790156504
version__ 
owner__dp dp
cc__dp cc__rgl
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="rgl's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|rgl commented|
|-|-|

```
#!html
<pre>
Don't add the https port if we are using the default one (443).
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez commented|
|-|-|

```
#!html
<pre>
Checking in twisted/web/server.py;
/cvs/Twisted/twisted/web/server.py,v  &lt;--  server.py
new revision: 1.105; previous revision: 1.104
done
Checking in twisted/test/test_web.py;
/cvs/Twisted/twisted/test/test_web.py,v  &lt;--  test_web.py
new revision: 1.41; previous revision: 1.40
done

Fixed.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub commented|
|-|-|

In changeset 0cf15cbd53ddab65388f21f3d60e1e86765545f2
```
#!CommitTicketReference repository="" revision="0cf15cbd53ddab65388f21f3d60e1e86765545f2"
Merge pull request #310 from twisted/8581-scop-files

Author: scop
Reviewer: rodrigc
Fixes: #8581

More file close related fixes
```
